setup.py

```diff
@@ -9,7 +9,7 @@
 setup(
     name="perceiver-model",
     packages=find_packages(),
-    version="0.7.2",
+    version="0.7.3",
     license="MIT",
     description="Multimodal Perceiver - Pytorch",
     author="Jacob Bieker, Jack Kelly, Peter Dudfield",
```

Comment:
Should we include lucidrains?!?

Reply:
Maybe we could add something in the README saying its originally from him, and we are extending it? Then it'll still be included in PyPi and such

Reply:
SGTM! :)